app/assets/javascripts/modules/WizardForm.js

```diff
@@ -0,0 +1,61 @@
+define(['jquery'], function($) {
+  'use strict';
+
+  var $form = $('[data-wizard-form] form'),
+      $formItems = $form.find('.wizard-form-item'),
+      $submitButton = $form.find('button.button--primary'),
+      $prevButton = $form.find('button.button--secondary'),
```

Comment:
we prefer to use `data-` values to find elements in the DOM instead of using classes, which should be reserved for styles.